optuna/study/study.py

```diff
@@ -222,6 +226,8 @@ def get_trials(
         """Return all trials in the study.
 
         The returned trials are ordered by trial number.
+        .. seealso::
```

Comment:
I suppose one empty line is necessary to fix the CI error.

```suggestion

        .. seealso::
```